src/Core/System/Snippet/Command/InstallTranslationCommand.php

```diff
@@ -112,4 +134,16 @@ private function createProgressBar(OutputInterface $output, int $count): Progres
 
         return $progressBar;
     }
+
+    private function saveMetadata(MetadataCollection $metadata, OutputInterface $output): void
+    {
+        $output->writeln('Loading translation metadata...');
```

Comment:
Why are the outputs “loading” when the metadata is being saved?

Reply:
good catch, will fix it